templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -892,29 +927,38 @@ export default {
       if (evt === "multi-content") this.node.presentationStyle = "accordion"
     },
     async setLinkData() {
-      if (shouldFetch(this.node.typeData.mediaURL, this.node)) {
+      if (
+        this.kaltura_info === null &&
+        shouldFetch(this.node.typeData.mediaURL, this.node)
+      ) {
         const url = this.node.typeData.mediaURL
         const { data } = await getLinkMetadata(url)
 
         if (data) {
           this.node.typeData.linkMetadata = data
-          if (
-            confirm(
-              "Would you like to use the link preview image as the thumbnail image?"
-            )
-          ) {
-            this.node.imageURL = data.image
-          }
-          if (
-            confirm(
-              "Would you like to use the link preview image as the locked thumbnail image?"
-            )
-          ) {
-            this.node.lockedImageURL = data.image
+
+          if (this.type === "add") {
```

Comment:
Hmm.. What if the user is changing the media? Should we not ask them in that case as well?